docs/markdown/Gnome-module.md

```diff
@@ -298,8 +298,20 @@ VAPI or Vala binaries.
 
 ### gnome.yelp()
 
+```meson
+  gnome.yelp(id: string, sources: ...string, sources: []string, media: []string,
+             languages: []string, symlink_media: bool = true): void
+```
+
 Installs help documentation using Yelp. The first argument is the
-project id.
+project id. Additionally it may take sources as variadic arguments.
```

Comment:
I would kinda prefer something like:

> Additionally, there was undocumented support for sources as positional arguments rather than as a kwarg. This should never be used, and if you did use it in combination with proper kwargs, then the positional arguments would be silently ignored.
>
> Since 0.60.0, this raises a warning and may be removed at any time.

And then just... don't remove it for a few releases.

Reply:
I just feel like documenting it as "valid but deprecated" is giving it too much credit and people might for some strange reason decide to use it because it works for earlier versions of meson. Emphasizing that it was never officially allowed and that we make zero commitment to keeping support for it (whereas deprecated things are usually kept around for a long time and are thus somehow "safe") will give people the additional impetus to stop using this really fast, if they happened to be doing so.

Reply:
I tried to update the comments to point out that this used to work, but we don't support it, and it's slated for removal.

Reply:
Nice, wording seems good.